docs/locale/fr/LC_MESSAGES/userguide/installation.po

```diff
@@ -118,81 +164,143 @@ msgstr ""
 msgid "macFUSE download screen"
 msgstr "écran de téléchargement de macFUSE"
 
-#: ../../userguide/installation.rst:59
+#: ../../userguide/installation.rst:89
+msgid "Open the ``.dmg`` file and follow its instructions to install."
+msgstr "Ouvrez le fichier ``.dmg`` et suivez les instructions d'installation."
+
+#: ../../userguide/installation.rst:91
 msgid ""
-"Open the ``.dmg`` file and follow its instructions to install. If the "
-"opening fails, two options are available in `System Settings > Privacy and "
-"Security`: either check the `App Store and identified developers` box, or "
-"click `Open Anyway` if you don't want to change this setting, which will "
-"need to be done once to open the ``.dmg``, and possibly once more to start "
-"the installer."
-msgstr ""
-"Ouvrez le ``.dmg`` et suivez les instructions d'installation. Si l'ouverture "
-"échoue, il y a deux solutions dans `Réglages du système > Confidentialité et "
-"sécurité` : soit activer l'autorisation pour `L'app store et les "
-"développeurs identifiés`, soit autoriser l'ouverture directement via le "
-"bouton `Ouvrir quand même` si vous ne souhaitez pas activer cette option. "
-"Dans ce deuxième cas, il faudra l'utiliser pour ouvrir le ``.dmg`` et "
-"possiblement une deuxième fois pour lancer l'installeur compris dedans."
+"If the opening fails, two options are available in `System Settings > Privacy "
+"and Security`: either check the `App Store and identified developers` box, or "
+"click `Open Anyway` if you don't want to change this setting, which will need "
+"to be done once to open the ``.dmg``, and possibly once more to start the "
+"installer."
+msgstr ""
+"Si l'ouverture échoue, deux options sont disponibles dans `System Settings > "
+"Privacy and Security` : cochez la case `App Store et développeurs "
+"identifiés`, ou cliquez sur `Open Anyway` si vous ne voulez pas changer ce "
+"paramètre, ce qui devra être fait une fois pour ouvrir le ``.dmg``, et "
+"éventuellement une fois de plus pour démarrer l'installateur."
```

Comment:
Maybe "Réglages du système > Confidentialité et sécurité` and `Ouvrir quand même` instead of `System Settings > Privacy and Security`  and `Open Anyway`